edgexfoundry/edgex-compose
# refactor: Adjust compose to fix changes breaking the TAF tests

This PR is dependent on https://github.com/edgexfoundry/edgex-go/pull/4308

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD


## Testing Instructions
run `make docker` using branch from https://github.com/edgexfoundry/edgex-go/pull/4308
run make run ds-virtual dev from compose builder
Verify all service bootstrap successfully
